kinta-cli/src/main/kotlin/com/dailymotion/kinta/Main.kt

```diff
@@ -35,22 +43,20 @@ val compiledCommands = listOf(
 )
 
 fun main(args: Array<String>) {
-    val hasKintaDir = File("kintaSrc").exists()
-
-    check(hasKintaDir
-            || args.isEmpty()
-            || compiledCommands.map { it.commandName }.contains(args[0])) {
-        "Kinta is not initialized. Please run 'kinta init' at the root of your project"
-    }
```

Comment:
This check was useless: by definition, there's no runtime command if there's no kintaSr.

Reply:
👍